src/components/enterprise-invite/index.js

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
q: should we rename some of these files to `index.js`?. Does not have to be in this pr.